### ISSUE TITLE
Vault Abilities dont show

### ISSUE THREAD

**Issue by Stevendoesstuff** [Iskallia/Vault-public-S1#57]
![2021-03-25_23 04 03](https://user-images.githubusercontent.com/81401903/112575597-9cdd8d00-8dbe-11eb-85bd-cf0e733198b1.png)

Vault Abilities do not show up, It,s just a black book.  I've generated multiple worlds and re-loaded the game files and it is the same every time.

**Comment by Nimothy** [comment]
Hi @Stevendoesstuff please could you provide the following information:

1. Which version of the pack are you on?
2. How did you install the pack?
3. Which Launcher are you using?

**Comment by Stevendoesstuff** [comment]
I’m using the newest version of the pack, I installed it from iskalls discord/GitHub onto curse forge launcher

**Comment by Nimothy** [comment]
Hi @Stevendoesstuff, my guess is the mod isn't launching correctly. 
It would be helpful to get the logs to see what's going on. 

You may find this helpful: https://github.com/Iskallia/Vault-public/issues/70#issuecomment-808148163

**Comment by Stevendoesstuff** [comment]
Will send when I return home

**Comment by Stevendoesstuff** [comment]
how do i get logs to send you?

**Comment by Nimothy** [comment]
@Stevendoesstuff there should be a logs folder in the instance folder for that instance of Minecraft. 
We've seen several issues with direct imports into curseforge. Did you try following the instructions in the linked comment?

**Comment by Stevendoesstuff** [comment]
@Nimothy  can i dm these to you?

**Comment by Stevendoesstuff** [comment]
I followed the instructions you mentioned.  I did it that way the first time i tried and i was having the same issue as a few others where the world would get to 100% and then not load.  I noticed others saying that using curse forge fixed it for them so i switched to curse forge where i had the problem with the vault abilities.  now after doing it with multimc again it still wont load after getting to 100%.

do i just have to wait for the official release?

**Comment by Nimothy** [comment]
@Stevendoesstuff, feel free to DM ChufnutII on discord.

**Comment by Nimothy** [comment]
@Stevendoesstuff let's use the official released pack: [vault-hunters-official-modpack]( https://www.curseforge.com/minecraft/modpacks/vault-hunters-official-modpack/files)
Hopefully, this works.

**Comment by Stevendoesstuff** [comment]
Yeah I booted it up quick this morning before work and was still having the same issue. I tried on both curse forge launcher and multiMC and had the same issue I was having before on both launchers

**Comment by Nimothy** [comment]
@jmilthedude might be an actual bug.

**Comment by Stevendoesstuff** [comment]
I loaded into a server last night on multiMC which is the first time I’ve actually gotten into a world through that launcher. I had the same issue through that launcher where the vault abilities screen is blacked out. Originally I was having the issue in curse forge launcher

**Comment by Stevendoesstuff** [comment]
has anyone been able to look into this?  im still having the issue
